### ISSUE TITLE
Add new tab and tag to Strapi for blogs

### ISSUE THREAD

**Issue by EstuaryRob** [estuary/marketing-site#412]
## Description

Want to add a new tag "Data Insights" and a new tab to in between All and Data Basics (as the new second tab) called "Data Insights"

Once the work is complete we'll start to move blogs over.

## Work required

-   [ ] engineering will fill this in

## Figma / Designs

include any screenshots/links of designs

**Comment by travjenkins** [comment]
Added. The ordering is not handled in Strapi and there is no support for that. That will require actual code changes and should be a stand alone task.

![image](https://github.com/user-attachments/assets/58860b1e-310b-4f54-b95c-56db8303964a)

**Comment by EstuaryRob** [comment]
I'll add a new issue then.
